### PR TITLE
[#IP-356] Update Node to 14.16.0

### PR DESCRIPTION
**Pull request by balanza**
Update Node to 14.16.0

### CODE REVIEW

**Comment by BurnedMarshal** [comment]
Should be changed the NODE_VERSION inside devops folder?
https://github.com/pagopa/io-ts-commons/blob/IP-356--update-node-version/.devops/code-review-pipelines.yml#L10
https://github.com/pagopa/io-ts-commons/blob/IP-356--update-node-version/.devops/deploy-pipelines.yml#L12

File: .devops/deploy-pipelines.yml [hunk 1]
```diff
@@ -60,15 +58,13 @@ stages:
       - job: make_release
         steps:
         - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/master') }}:
-          - template: templates/node-github-release/template.yaml@pagopaCommons 
+          - template: templates/node-github-release/template.yaml@pagopaCommons
```
Comment (balanza) [inline .devops/deploy-pipelines.yml:61]:
```suggestion
          - template: templates/node-job-setup/template.yaml@pagopaCommons
            parameters:
              persistCredentials: true
          - template: templates/node-github-release/template.yaml@pagopaCommons
```

File: .devops/deploy-pipelines.yml [hunk 2]
```diff
@@ -60,15 +58,16 @@ stages:
       - job: make_release
         steps:
         - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/master') }}:
-          - template: templates/node-github-release/template.yaml@pagopaCommons 
+          - template: templates/node-job-setup/template.yaml@pagopaCommons
             parameters:
+              persistCredentials: true
+          - template: templates/node-github-release/template.yaml@pagopaCommons
+            parameters:
+              persistCredentials: true
```
Comment (BurnedMarshal) [inline .devops/deploy-pipelines.yml:66]:
```suggestion
```

File: yarn.lock [hunk 3]
```diff
@@ -2294,9 +2295,9 @@ form-data@~2.3.2:
     mime-types "^2.1.12"
 
 fp-ts@^2.10.5:
-  version "2.10.5"
-  resolved "https://registry.yarnpkg.com/fp-ts/-/fp-ts-2.10.5.tgz#7c77868fe8bd9b229743303c1bec505b959f631b"
-  integrity sha512-X2KfTIV0cxIk3d7/2Pvp/pxL/xr2MV1WooyEzKtTWYSc1+52VF4YzjBTXqeOlSiZsPCxIBpDGfT9Dyo7WEY0DQ==
+  version "2.11.1"
+  resolved "https://registry.yarnpkg.com/fp-ts/-/fp-ts-2.11.1.tgz#b1eeb2540728b6328542664888442f8f805d2443"
+  integrity sha512-CJOfs+Heq/erkE5mqH2mhpsxCKABGmcLyeEwPxtbTlkLkItGUs6bmk2WqjB2SgoVwNwzTE5iKjPQJiq06CPs5g==
```
Comment (BurnedMarshal) [inline yarn.lock:2300]: Can we use any version of fp-ts? Seems that 2.11.x add a lot of changes

Comment (balanza) [reply yarn.lock:2300]: I assume as it builds and tests pass...

**Review by BurnedMarshal — APPROVED** [review]

**Pull request merged**